### ISSUE TITLE
Django migrations not run on upgrade

### ISSUE THREAD

**Issue by paddatrapper** [libretime/libretime#1487]
The Django database migrations are run as part of the installation web form when the database is created. However, the migrations are not run on upgrade, as the setup is not run then, nor when the config file is pre-populated. In these cases, currently the migrations must be run manually using

```
LIBRETIME_CONF_FILE=/etc/airtime/airtime.conf.temp libretime-api migrate
```

This should run in the installer if the database exists and can be accessed (i.e. on upgrade, but not fresh install).

**Comment by jooola** [comment]
I guess this is mainly a documentation problem, according to #1543 we agreed that the migration should be handled by hand anyway. I think it safe to write a extra step in the install/upgrade guide to run the api migrations.

**Comment by jooola** [comment]
Fixed in #1687
